### PR TITLE
Update GOVERNANCE.md

### PR DESCRIPTION
**Pull request by krobinsonca**

### CODE REVIEW

File: GOVERNANCE.md
Comment (swcurran) [inline GOVERNANCE.md:17]: So the idea is to add the list of people involved with the Governing Authority in the MAINTAINERS file, in addition to the Editors, right?  Two groups will be listed?

Comment (krobinsonca) [reply GOVERNANCE.md:17]: As our governance model evolves, we can adjust the governance document. But for now, we need to just state how things work, today. Today, essentially, the Editors are the ones making decisions on PRs and Merges without external direction. If external direction becomes "a thing", then we can update the document to describe the process.

**Comment by swcurran** [comment]
I disagree. The Editors will merge things as long as they are comfortable, and if not — escalate the issue.  For now — that’s to John, Jill, etc.  But I do thing the “escalation group” should be named, or at least an email address to contact for disputes.

How the “governing authority” works internally can be opaque, IMHO.

**Comment by krobinsonca** [comment]
Agreed. The main point I was trying to make is to describe the current state. If our current state is Editors follow internal escalation, then we simply say that, no details on who. Maybe just the "Digital Trust Executive Team". 
For disputes, we can put in a contact email address

**Comment by swcurran** [comment]
Agreed. Like the idea of putting the reference to the group and contact point in the MAINTAINERS.md file.

File: GOVERNANCE.md
Comment (swcurran) [reply GOVERNANCE.md:17]: I missed this comment.  I think we have external direction now — per notes below.

**Review by swcurran — APPROVED** [review]

**Pull request merged**
